vulnerabilities/importers/fireeye.py

```diff
@@ -0,0 +1,139 @@
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# VulnerableCode is a trademark of nexB Inc.
+# SPDX-License-Identifier: Apache-2.0
+# See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+# See https://github.com/nexB/vulnerablecode for support or download.
+# See https://aboutcode.org for more information about nexB OSS projects.
+#
+import os
+import re
+from typing import Iterable
+
+from vulnerabilities.importer import AdvisoryData
+from vulnerabilities.importer import GitImporter
+from vulnerabilities.importer import Reference
+from vulnerabilities.utils import build_description
+from vulnerabilities.utils import dedupe
+
+
+class FireyeImporter(GitImporter):
+    spdx_license_expression = "CC-BY-SA-4.0 AND MIT"
+    license_url = "https://github.com/mandiant/Vulnerability-Disclosures/blob/master/README.md"
+
+    def __init__(self):
+        super().__init__(repo_url="git+https://github.com/mandiant/Vulnerability-Disclosures")
+
+    def advisory_data(self, _keep_clone=True) -> Iterable[AdvisoryData]:
+        self.clone()
+        for root_dir in os.listdir(self.vcs_response.dest_dir):
+            if root_dir in ("README.md", ".git"):
+                continue
+
+            for root, _, files in os.walk(os.path.join(self.vcs_response.dest_dir, root_dir)):
+                for file in files:
+                    if file.endswith((".md", ".MD")):
+                        with open(os.path.join(root, file), "r", encoding="ISO-8859-1") as f:
```

Comment:
Why using "ISO-8859-1"?  Also I wonder if you could use pathlib and glob instead? https://docs.python.org/3/library/pathlib.html#pathlib.Path.glob

Reply:
without ISO-8859-1 I get this error 
`UnicodeDecodeError: 'utf-8' codec can't decode byte 0x96 in position 290: invalid start byte
`https://stackoverflow.com/questions/45529507/unicodedecodeerror-utf-8-codec-cant-decode-byte-0x96-in-position-35-invalid